src/components/AppInfoTooltipContent/ApeInfoTooltipContentToggle.tsx

```diff
@@ -0,0 +1,68 @@
+import { Link } from 'react-router-dom';
+
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(() => ({
+  container: {
+    width: '224px',
+    color: '#5E6F74',
+    fontSize: '14px',
+    fontStyle: 'normal',
+    lineHeight: '20px',
+    letterSpacing: '0em',
+    textAlign: 'left',
+  },
+
+  contentType: {
+    textTransform: 'capitalize',
+    fontSize: '16px',
+  },
+
+  link: {
+    textAlign: 'center',
+    color: '#00ABBF',
+  },
+}));
+export const ApeInfoTooltipContentForToggle = ({
+  yesContent,
+  noContent,
+  documentLink,
+}: {
+  yesContent: string;
```

Comment:
A general pattern could be:

[['yes', 'yesContent'], ['no', 'noContent']]

There is also what is being used for `ProfileCard`, I just did one version there, but if we have a few, than we could have an abstraction that encompasses multiple uses.

Reply:
and it could accept either the array of pairs or a react node or a string and also the link and link text. All of that could be added to the existing tooltip instead of a new component